### ISSUE TITLE
[Bug]: Ownership transfer throws notification "Transfer refused" even if Accept button is clicked. But the transfer is done.

### ISSUE THREAD

**Issue by Mer0me** [nextcloud/server#47042]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When you accept an ownership transfer, the owner receive a notification claiming you have refused the transfer.
However, the transfer is done a few minutes later.

### Steps to reproduce

1. Go to User's Parameter page
2. Choose a file or folder to transfer to someone
3. Let someone accept the transfer through his web interface
4.  Observe your notifications

### Expected behavior

Notification should display "Transfer accepted"

### Installation method

Community Web installer on a VPS or web space

### Nextcloud Server version

29

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated from a MINOR version (ex. 28.0.1 to 28.0.2)

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [X] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "ratelimit.protection.enabled": false,
        "auth.bruteforce.protection.enabled": false,
        "simpleSignUpLink.shown": false,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.agglo-montbeliard.fr",
            "192.168.50.106",
            "192.168.50.106"
        ],
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/cloud.agglo-montbeliard.fr\/",
        "dbtype": "mysql",
        "version": "29.0.4.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "Europe\/Paris",
        "installed": true,
        "theme": "",
        "loglevel": 3,
        "debug": false,
        "maintenance": false,
        "maintenance_window_start": 1,
        "app.mail.background-sync-interval": 86400,
        "app.mail.verify-tls-peer": false,
        "activity_expire_days": 365,
        "trashbin_retention_obligation": "auto, 15",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "app.mail.imap.timeout": 620,
        "appstore.experimental.enabled": true,
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "\\OCA\\User_LDAP\\LDAPProviderFactory",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_sendmailmode": "smtp",
        "mail_smtpport": "25",
        "mysql.utf8mb4": true,
        "default_phone_region": "FR",
        "has_rebuilt_cache": true,
        "app_install_overwrite": [
            "spreed",
            "files_mindmap",
            "pdfdraw",
            "issuetemplate",
            "files_linkeditor",
            "files_texteditor",
            "files_trackdownloads",
            "emlviewer",
            "files_archive",
            "impersonate",
            "extract",
            "tables",
            "cfg_share_links",
            "workflow_script",
            "files_emlviewer"
        ],
        "allow_local_remote_servers": true,
        "default_language": "fr",
        "default_locale": "fr_FR",
        "force_locale": "fr_FR",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "preview_max_memory": 512,
        "twofactor_enforced": "false",
        "twofactor_enforced_groups": [],
        "twofactor_enforced_excluded_groups": [],
        "sharing.enable_share_mail": false,
        "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
        "bulkupload.enabled": "false"
    }
}
```

### List of activated Apps

```shell
Enabled:
  - activity: 2.21.1
  - admin_audit: 1.19.0
  - analytics: 4.14.0
  - announcementcenter: 6.8.1
  - calendar: 4.7.14
  - cfg_share_links: 5.1.0
  - circles: 29.0.0-dev
  - cloud_federation_api: 1.12.0
  - comments: 1.19.0
  - contacts: 6.0.0
  - contactsinteraction: 1.10.0
  - dashboard: 7.9.0
  - dav: 1.30.1
  - deck: 1.13.1
  - drawio: 3.0.2
  - external: 5.4.0
  - extract: 1.3.6
  - federatedfilesharing: 1.19.0
  - federation: 1.19.0
  - files: 2.1.0
  - files_accesscontrol: 1.19.1
  - files_automatedtagging: 1.19.0
  - files_downloadlimit: 2.0.0
  - files_emlviewer: 2.9.0
  - files_external: 1.21.0
  - files_linkeditor: 1.1.20
  - files_lock: 29.0.1
  - files_mindmap: 0.0.30
  - files_msgviewer: 2.9.0
  - files_reminders: 1.2.0
  - files_sharing: 1.21.0
  - files_texteditor: 2.15.1
  - files_trashbin: 1.19.0
  - files_versions: 1.22.0
  - files_zip: 1.6.0
  - forms: 4.2.4
  - groupfolders: 17.0.1
  - guests: 3.1.0
  - impersonate: 1.16.0
  - integration_giphy: 1.0.11
  - integration_openai: 2.0.3
  - integration_replicate: 3.0.0
  - logreader: 2.14.0
  - lookup_server_connector: 1.17.0
  - mail: 3.7.6
  - nextcloud_announcements: 1.18.0
  - notifications: 2.17.0
  - notify_push: 0.7.0
  - oauth2: 1.17.0
  - onlyoffice: 9.3.0
  - password_policy: 1.19.0
  - passwords: 2024.7.10
  - photos: 2.5.0
  - polls: 7.1.4
  - provisioning_api: 1.19.0
  - serverinfo: 1.19.0
  - settings: 1.12.0
  - spreed: 19.0.7
  - systemtags: 1.19.0
  - tables: 0.7.5
  - tasks: 0.16.0
  - text: 3.10.1
  - text_templates: 1.1.0
  - theming: 2.4.0
  - theming_customcss: 1.17.0
  - thesearchpage: 1.2.7
  - twofactor_backupcodes: 1.18.0
  - twofactor_totp: 11.0.0-dev
  - updatenotification: 1.19.1
  - user_ldap: 1.20.0
  - user_status: 1.9.0
  - user_usage_report: 1.13.1
  - viewer: 2.3.0
  - weather_status: 1.9.0
  - workflow_pdf_converter: 1.14.0
  - workflowengine: 2.11.0
Disabled:
  - appointments: 1.15.2 (installed 1.15.2)
  - approval: 1.0.12 (installed 1.0.12)
  - auto_groups: 1.4.1 (installed 1.4.1)
  - bruteforcesettings: 2.9.0 (installed 2.2.0)
  - cadviewer: 8.75.4 (installed 8.75.4)
  - calendar_resource_management: 0.1.0 (installed 0.1.0)
  - customproperties: 2.0.4 (installed 2.0.4)
  - duplicatefinder: 1.0.5 (installed 1.0.5)
  - emlviewer: 1.0.9 (installed 1.0.9)
  - encryption: 2.17.0
  - event_update_notification: 2.0.0 (installed 2.0.0)
  - files_archive: 1.1.2 (installed 1.1.2)
  - files_downloadactivity: 1.16.0 (installed 1.16.0)
  - files_markdown: 2.3.6 (installed 2.3.6)
  - files_mindmap.old: 0.0.30
  - files_pdfviewer: 2.10.0 (installed 2.10.0)
  - files_rightclick: 0.15.1 (installed 1.6.0)
  - files_trackdownloads: 1.11.0 (installed 1.11.0)
  - firstrunwizard: 2.18.0 (installed 2.12.0)
  - group_everyone: 0.1.10 (installed 0.1.10)
  - hibp: 0.1.0 (installed 0.1.0)
  - integration_deepl: 1.0.0 (installed 1.0.0)
  - integration_ews: 1.0.37 (installed 1.0.37)
  - issuetemplate: 0.7.0 (installed 0.7.0)
  - mailnotifier: 0.0.3 (installed 0.0.3)
  - pdfdraw: 0.1.2
  - privacy: 1.13.0 (installed 1.7.0)
  - ransomware_protection: 1.14.0 (installed 1.14.0)
  - recommendations: 2.1.0 (installed 1.3.0)
  - related_resources: 1.4.0 (installed 1.1.0-alpha1)
  - sharebymail: 1.19.0 (installed 1.17.0)
  - smb_test: 0.3.5 (installed 0.3.5)
  - snappymail: 2.36.3 (installed 2.36.3)
  - support: 1.12.0 (installed 1.6.0)
  - survey_client: 1.17.0 (installed 1.11.0)
  - suspicious_login: 7.0.0
  - workflow_script: 1.11.1 (installed 1.11.1)
```

### Nextcloud Logs

```shell
no relevant log
```

**Comment by DanielLBarlow** [comment]
I too had this same issue as well. But i also found that if i left it a while that i would then get a transfer success notification after. As well as the file or folder being transfered before this message as well

**Comment by hafiztahirFT** [comment]
facing same issue with  29.0.7

**Comment by joshtrichards** [comment]
Fixed in #48734
